packages/template-retail-react-app/app/components/experience/image-with-text/index.jsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React, {useEffect, useState} from 'react'
+import PropTypes from 'prop-types'
+import {Image, Box, Text, Link} from '@chakra-ui/react'
+import DOMPurify from 'dompurify'
+
+/**
+ * Image with text component
+ *
+ * @param ITCLink Image Link.
+ * @param ITCText Text Below Image.
+ * @param image Image.
+ * @param heading Text Overlay.
+ * @param alt The image alt text shown by the component.
+ * @returns {JSX.Element}
```

Comment:
```suggestion
 * @param {object} props
 * @param {string} - props.ITCLink - Image Link.
 * @param {string} - props.ITCText - Text Below Image.
 * @param {image} - props.image - Image.
 * @param {string} - props.heading - Text Overlay.
 * @param {string} - props.alt - The image alt text shown by the component.
 * @returns {React.ReactElement} - ImageWithText component.
```

For consistency sake, lets make all our JSDOC definitions look like they were done by the same developer.